code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -195,20 +195,27 @@ Contains:
 	item_state = "ert_security"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/sec
 	tauric = TRUE		//Citadel Add for tauric hardsuits
+	armor = list("melee" = 75, "bullet" = 80, "laser" = 80, "energy" = 80, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 80)

```

Comment:
:thinking:

Reply:
armor = list("melee" = 75, "bullet" = 60, "laser" = 60, "energy" = 60, "bomb" = 55, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 80)
was the armor it was MENT to be